docs/plone-deployment/edit-project.md

```diff
@@ -28,65 +28,91 @@ The Plone community maintains a curated list of Plone Backend and Plone Classic
 
 ## Starting the servers
 
-After a [new project](new-project.md) was created lets check every thing runs as expected.
+After a [new project](new-project) was created, let's check that everything runs as expected.
+
 
 ### Frontend
-On a terminal, run the following code to start the frontend server:
 
-```{code-base} shell
+In a terminal, run the following code to start the frontend server:
+
+```shell
 make start-frontend
 ```
+
 After a while you should see:
 
-TODO: Add image here
+```console
+yarn run v1.22.19
+$ razzle start
+ WAIT  Compiling...
+
+
+✔ Client
+  Compiled successfully in 620.55ms
+
+✔ Server
+  Compiled successfully in 25.64s
+
+✅  Server-side HMR Enabled!
+sswp> Handling Hot Module Reloading
+Volto is running in SEAMLESS mode
+Using internal proxy: http://localhost:3000 -> http://localhost:8080/Plone
+🎭 Volto started at 0.0.0.0:3000 🚀
+```
 
 ### Backend
 
 On a second terminal, run the following code to start the backend server:
 
-```{code-base} shell
-make start-frontend
+```shell
+make start-backend
 ```
 
 After a while you should see:
 
-```
-INFO    [waitress:486][MainThread] Serving on http://0.0.0.0:8080`
+```console
+INFO    [waitress:486][MainThread] Serving on http://0.0.0.0:8080
 ```
 
-## Check Local access
+## Check local access
+
+Visit the Plone site at the following URL:
 
 http://plone-conference.localhost:3000
 
+
 ## Stopping the servers
 
-On both terminals press `Ctrl-c`.
+In both terminals, press {kbd}`Ctrl-C`.
+
 
-# Adding OAuth Support
+## Adding OAuth support
 
-As an example for this training we add a new authentication method:
-Login with GitHub.
+As an example of adding an add-on for this training, we will add a new authentication method, login with GitHub.
 
-There is already a plugin available consisting out of one module for Plone Frontend and another for Plone Backend.
+There is already a plugin available consisting of one module for Plone Frontend and another for Plone Backend.
 
-The Plugin needs am application key from Github.
+The Plugin needs an application key from GitHub.
 
-First you need to create a new application on GitHub ...
+First you need to create a new application on GitHub.
```

Comment:
We should link to GitHub's documentation for this (I went to look for it, then realized I'm not sure whether the training intends the use of a Github Oauth App or a Github App. Probably either could work, because the latter supports use OAuth as well.)

Reply:
https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app or https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app

Reply:
Yes, I am still not sure myself. I will walk through the process using the OAuth docs, and see if that will do.

Reply:
I meant a Github App here.

Reply:
I pushed a commit with more changes. However, I was not able to OAuth, so these instructions probably need to be checked against a working example. Help appreciated.

Reply:
> I meant a Github App here.

That one has a lot more steps than the OAuth one. Should we use the simpler OAuth one instead?

Reply:
Well, that's the thing. I don't know the correct values to use, so I made a best guess. I pushed a commit recently with what I tried, but it does not work.

See https://deploy-preview-691--plone-training.netlify.app/plone-deployment/edit-project#get-your-github-key-and-secret and the next section.

Reply:
Additionally should I be using the Volto or Classic UI to login with the plugin?